sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePlugin.scala

```diff
@@ -38,11 +51,11 @@ object ScalaNativePlugin extends AutoPlugin {
       settingKey[String]("Compilation mode, either \"debug\" or \"release\".")
```

Comment:
Both `nativeCompileOptions` and `nativeLinkingOptions` might do environment discovery. This is still not addressed in this PR.